src/nvim/message.c

```diff
@@ -2438,7 +2438,7 @@ static void redir_write(char_u *str, int maxlen)
     if (*s != '\n' && *s != '\r') {
       while (cur_col < msg_col) {
         if (redir_reg)
-          write_reg_contents(redir_reg, (char_u *)" ", -1, TRUE);
+          write_reg_contents(redir_reg, (char_u *)" ", STRLEN(" "), true);
```

Comment:
For static strings there is `sizeof(" ") - 1`. Though I have no idea why you can’t just write `1` here, I would prefer this. (Also no need to use `STRLEN` in case when argument is `char*`: do not know others’ opinion, but I tend to avoid uneeded macros and use `strlen`.)